### PR TITLE
Blog post about the 1st workshop

### PR DESCRIPTION
**Pull request by bebatut**
@willdurand can you check the format?
@bgruening @mmiladi can you check the content?
Thanks both!

### CODE REVIEW

File: _events/2018-11-10-first_workshop.md [hunk 1]
```diff
@@ -1,16 +1,138 @@
...
 tags: [talk, training]
-title: 'Test of DNA extraction and sequencing'
+title: 'BeerDEcoded: unofficial kickoff'
+description: 'November, 10th, 2018'
 starts: 2019-11-10
 ends: 2019-11-10
+tweet: 'https://twitter.com/i/moments/1068433531829932032?ref_src=twsrc%5Etfw'
 location:
   city: Freiburg
   region: Baden Würtemberg
```
Comment (bgruening) [inline _events/2018-11-10-first_workshop.md]: Baden-Württemberg

File: _events/2018-11-10-first_workshop.md [hunk 2]
```diff
@@ -1,16 +1,138 @@
...
+    image1_path="/images/events/2018-11-10-first_workshop/DSC_6931.jpg"
+    image2_path="/images/events/2018-11-10-first_workshop/DSC_6878.jpg"
+    image3_path="/images/events/2018-11-10-first_workshop/DSC_6852.jpg"
+    image4_path="/images/events/2018-11-10-first_workshop/DSC_6930.jpg"
+    image5_path="/images/events/2018-11-10-first_workshop/DSC_6923.jpg"
+%}
+
+## DNA extraction
+
+We used a Chimay red and wanted to extract the DNA of the yeast inside this beer. To do so, we:
```
Comment (bgruening) [inline _events/2018-11-10-first_workshop.md]: link? http://chimay.com/bieres/rouge/

File: _events/2018-11-10-first_workshop.md [hunk 3]
```diff
@@ -1,16 +1,138 @@
...
+  image4_path="/images/events/2018-11-10-first_workshop/DSC_7002.jpg"
+  image5_path="/images/events/2018-11-10-first_workshop/DSC_7001.jpg"
+  image6_path="/images/events/2018-11-10-first_workshop/DSC_7006.jpg"
+  image7_path="/images/events/2018-11-10-first_workshop/DSC_7007.jpg"
+  image8_path="/images/events/2018-11-10-first_workshop/DSC_7009.jpg"
+%}
+
+At the end we quickly check the purity with a spectrophotometer and freezed the DNA until library preparation
+
+*The details of the protocol we used can be found in this website: [DNA extraction protocol]({% link _protocols/beer-dna-extraction.md %}).*
```
Comment (bgruening) [inline _events/2018-11-10-first_workshop.md]: can be found [here]()

File: _events/2018-11-10-first_workshop.md [hunk 4]
```diff
@@ -1,16 +1,138 @@
...
+
+## DNA sequencing
+
+During the DNA extracted, we started to play with the MinION with the installation of the Nanopore software and the test of flowcell. We then did the library preparation: preparation of the DNA for sequencing.
```
Comment (bgruening) [inline _events/2018-11-10-first_workshop.md]: During the DNA extraction, ...

and a functional test of the flowcell.

File: _events/2018-11-10-first_workshop.md [hunk 5]
```diff
@@ -1,16 +1,138 @@
...
+  url: https://www.flickr.com/photos/134305289@N03/shares/p58Xp0
 ---
 
-Today, we had an unofficial kickoff of the project. We poured Beer into the
-MinION's pores: sequencing yeast extracted from Beer with Nanopore.
+Today, we had an unofficial kickoff of the [BeerDEcoded project]({% link _projects/beerdecoded.md %}). 
+We poured Beer into the MinION's pores: sequencing yeast extracted from Beer with Nanopore.
+
+During this day, we reproduced and improved the [DNA extraction protocol]({% link _protocols/beer-dna-extraction.md %})
+that Stephan has developed. We also used for the 1st time our MinION. The results are very
```
Comment (bgruening) [inline _events/2018-11-10-first_workshop.md]: link to MinION? https://nanoporetech.com/products/minion

File: _events/2018-11-10-first_workshop.md [hunk 6]
```diff
@@ -1,16 +1,138 @@
...
+During the DNA extracted, we started to play with the MinION with the installation of the Nanopore software and the test of flowcell. We then did the library preparation: preparation of the DNA for sequencing.
+
+{% include four_central_image.html
+  image1_path="/images/events/2018-11-10-first_workshop/DSC_6857.jpg"
+  image2_path="/images/events/2018-11-10-first_workshop/DSC_6954.jpg"
+  image3_path="/images/events/2018-11-10-first_workshop/DSC_6853.jpg"
+  image4_path="/images/events/2018-11-10-first_workshop/DSC_6941.jpg"
+%}
+
+We then loaded the MinION with our DNA and launch the sequencing
```
Comment (bgruening) [inline _events/2018-11-10-first_workshop.md]: launched

File: _events/2018-11-10-first_workshop.md [hunk 7]
```diff
@@ -1,16 +1,138 @@
...
+
+{% include four_central_image.html
+  image1_path="/images/events/2018-11-10-first_workshop/DSC_6951.jpg"
+  image2_path="/images/events/2018-11-10-first_workshop/DSC_6946.jpg"
+  image3_path="/images/events/2018-11-10-first_workshop/DSC_6973.jpg"
+  image4_path="/images/events/2018-11-10-first_workshop/DSC_6981.jpg"
+%}
+
+It was quite exciting. We had long read until 24Kb, but the quality
```
Comment (bgruening) [inline _events/2018-11-10-first_workshop.md]: we had longs read__s__ or we had a long read

File: _events/2018-11-10-first_workshop.md [hunk 8]
```diff
@@ -1,16 +1,138 @@
...
+{% include four_central_image.html
+  image1_path="/images/events/2018-11-10-first_workshop/DSC_6951.jpg"
+  image2_path="/images/events/2018-11-10-first_workshop/DSC_6946.jpg"
+  image3_path="/images/events/2018-11-10-first_workshop/DSC_6973.jpg"
+  image4_path="/images/events/2018-11-10-first_workshop/DSC_6981.jpg"
+%}
+
+It was quite exciting. We had long read until 24Kb, but the quality 
+started to decrease quite quickly (probably not enough DNA) so we stopped
+it earlier.
```
Comment (bgruening) [inline _events/2018-11-10-first_workshop.md]: it early after our lunch break.

File: _events/2018-11-10-first_workshop.md [hunk 9]
```diff
@@ -1,16 +1,138 @@
...
+
+*The details of the protocol we used can be found in this website: [DNA sequencing protocol]({% link _protocols/beer-dna-sequencing.md %}).*
```
Comment (bgruening) [inline _events/2018-11-10-first_workshop.md]: redundant ... or same changes as above.

File: _events/2018-11-10-first_workshop.md [hunk 10]
```diff
@@ -1,16 +1,138 @@
...
+using simpler and everyday life tools, and ensure its robustness on every location and 
+situation, i.e., the protocol has to work in a specialised lab as well as in a high 
+school chemistry/biology labs. To overcome this challenge, we will collect ideas from 
+Do-It-Yourself biology projects.
+
+We will make special efforts in making the protocols accessible and understandable 
+by everyone, with graphics supporting each step and explaining their importance and 
+their execution. 
+
+For the data analysis, a dedicated workflow will be available via [Galaxy]().
```
Comment (bgruening) [inline _events/2018-11-10-first_workshop.md]: ping @mmiladi

Comment (bgruening) [reply _events/2018-11-10-first_workshop.md]: I would also link a history so we can share the data.

File: _events/2018-11-10-first_workshop.md [hunk 11]
```diff
@@ -1,16 +1,138 @@
...
+
+We will make special efforts in making the protocols accessible and understandable 
+by everyone, with graphics supporting each step and explaining their importance and 
+their execution. 
+
+For the data analysis, a dedicated workflow will be available via [Galaxy](). 
+This open source, web-based platform makes reproducible computational biology and 
+bioinformatics accessible to anyone via a web-browser. 
+
+A dedicated Galaxy server ([streetscience.usegalaxy.eu](https://streetscience.usegalaxy.eu/))
```
Comment (bgruening) [inline _events/2018-11-10-first_workshop.md]: should we link it if it does not yet exist?

File: _events/2018-11-10-first_workshop.md [hunk 12]
```diff
@@ -1,16 +1,138 @@
...
+
+A dedicated Galaxy server ([streetscience.usegalaxy.eu](https://streetscience.usegalaxy.eu/)) 
+will be implemented to serve the workflow. In complement, training material for 
+data analysis will be created in cooperation with the [Galaxy Training community](https://galaxyproject.github.io/training-material/) 
+with slides, hands-on material, interactive web-tours suitable to guide our participants 
+from data upload to data analysis. 
+
+As the aim of this project is to share our knowledge with the public, we will 
+organize two-day "DNA extraction, sequencing and data analysis" workshops, 
+one in March at a local biotechnology high school, and one in July for a larger audience.
```
Comment (bgruening) [inline _events/2018-11-10-first_workshop.md]: I would not mention time and targets at this point.

File: _events/2018-11-10-first_workshop.md (hunk 11 above)
Comment (bebatut) [reply _events/2018-11-10-first_workshop.md]: It will exist soon

File: _events/2018-11-10-first_workshop.md (hunk 10 above)
Comment (mmiladi) [reply _events/2018-11-10-first_workshop.md]: Here is the link: https://usegalaxy.eu/u/milad/h/nanoporebeerdecodechimaytriple

File: _events/2018-11-10-first_workshop.md (hunk 12 above)
Comment (mmiladi) [reply _events/2018-11-10-first_workshop.md]: :+1:

File: _events/2018-11-10-first_workshop.md (hunk 10 above)
Comment (bgruening) [reply _events/2018-11-10-first_workshop.md]: Merci!

**Comment by bebatut** [comment]
I merge it and @willdurand is working on the formatting on another PR

**Pull request merged**